test/zdtm/static/ghost_multi_hole01.desc

```diff
@@ -0,0 +1 @@
+{'dopts': '--ghost-limit 8M --ghost-fiemap'}
```

Comment:
Heh, as we have FIEMAP_DEFAULT true, we probably wanted to have --no-ghost-fiemap here.

Reply:
Indeed. Thanks for pointing out.
Fixed.